samples/graph-meeting-notification/nodejs/README.md

```diff
@@ -0,0 +1,127 @@
+---
+page_type: sample
+description: This is a sample application which demonstrates the use of online meeting subscription and sends you the notifications in chat using bot.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- nodejs
+extensions:
+ contentType: samples
+ createdDate: "24-09-2022 00:16:45"
+urlFragment: officedev-microsoft-teams-samples-graph-meeting-notification-nodejs
+---
+
+# Online meeting subscription
+
+This is a sample application which demonstrates use of online meeting subscription that will post notifications when user joined/left and when meeting start/end.
+
+## Concepts introduced in this sample
+- After sucessfully installation of bot in meeting you will get a welcome card and the subscription will be created for meeting it is installed in.
+
+![Welcome Card](Images/WelcomeCard.png)
+
+- After that when the metting gets started or user joins the meeting, notifications will be sent in chat:
+
+![Meeting Started](Images/MeetingStartedCard.png)
```

Comment:
Take a Teams full width screenshots for all the images and update at all the places. 
Also, .gif has the same full Teams width screenshots to make all the samples images consistent.